strapi/strapi
# fix: Add button of repeatable components is missing a focus outline and focus handling

### What does it do?

Adds outline on `Add button`  of repeatable components once focused.

### Why is it needed?
The `overflow: hidden` in `AccordionFooter` hides the outline, removing it allows the outline ring to show when focused.


### How to test it?
1. Create a content-type with a repeatable component
2. Tab to "Add an entry"
3. Notice that the button has a focus outline

### Demo


https://user-images.githubusercontent.com/1501599/208118888-76209f19-2736-460f-9324-86a2b7842fd6.mov



### Related Issue
https://github.com/strapi/strapi/issues/15109
